src/applications/disability-benefits/all-claims/tests/data/secondary-new-test.json

```
@@ -131,7 +132,7 @@
       {
         "cause": "SECONDARY",
         "view:secondaryFollowUp": {
-          "causedByDisability": "De-selected",
+          "causedByDisability": "Diabetes Mellitus0",
```

Looks like the idea here was to select the other disability. What do you think about changing this to the following?
```suggestion
          "causedByDisability": "Diabetes Mellitus1",
```